store/sqlstore/post_store_test.go

```diff
@@ -17,3 +18,23 @@ func TestPostStore(t *testing.T) {
 func TestSearchPostStore(t *testing.T) {
 	StoreTestWithSearchTestEngine(t, searchtest.TestSearchPostStore)
 }
+
+func TestMysqlStopWords(t *testing.T) {
```

Comment:
Nice addition. You have to include hyphenated words :)

Reply:
Ok